test/integration/index_documents_test.rb

```diff
@@ -0,0 +1,71 @@
+require "integration_test_helper"
+require 'govuk_message_queue_consumer'
+require 'govuk_message_queue_consumer/test_helpers'
+require './lib/index_documents'
+
+class IndexDocumentsTest < IntegrationTest
+  include IndexDocumentsTestHelpers
+
+  def setup
+    stub_elasticsearch_settings
+    create_test_indexes
+    stub_tagging_lookup
+    stub_calls_for_index_documents_test
+  end
+
+  def teardown
+    clean_test_indexes
+  end
+
+  def test_topics_get_indexed_from_publishing_api
+    Elasticsearch::Amender.any_instance.expects(:amend)
```

Comment:
I was operating under the assumption that other tests make sure that amend is called for the correct document, but fair enough, i'll add extra expectations on `amend`